interface/patient_file/summary/immunizations.php

```diff
@@ -343,497 +346,474 @@ function saveImmunizationObservationResults($id, $immunizationdata)
+                                            </div>
+                                        </div>
+                                <?php } ?>
+                            </div>
+
+                            <div class="row">
+                                <div class="col-12 text-center">
+                                    <button type="button" class="btn btn-primary btn-sm btn-add" onclick="addNewRow();" title='<?php echo xla('Click here to add new row'); ?>'>
+                                        <?php echo xlt('Add'); ?>
+                                    </button>
+                                </div>
+                            </div>
+
+                            <input type ="hidden" name="tr_count" id="tr_count" value="<?php echo (!empty($imm_obs_data) && count($imm_obs_data) > 0) ? attr(count($imm_obs_data)) : 1 ;?>" />
+                            <input type="hidden" id="clickId" value="" />
+                        </fieldset>
+                    </div>
+                    <div class="btn-group mt-3">
+                        <button type="button" class="btn btn-primary btn-save" name="save" id="save" value="<?php echo xla('Save Immunization'); ?>">
+                            <?php echo xlt('Save Immunization'); ?>
+                        </button>
+                        <button type="button" class="btn btn-primary btn-print" name="print" id="print" value="<?php echo attr(xl('Print Record') . ' (' . xl('PDF') . ')'); ?>">
+                            <?php echo xlt('Print Record (PDF)'); ?>
+                        </button>
+                        <button type="button" class="btn btn-primary btn-print" name="printHtml" id="printHtml" value="<?php echo attr(xl('Print Record') . ' (' . xl('HTML') . ')'); ?>">
+                            <?php echo xlt('Print Record (HTML)'); ?>
+                        </button>
+                        <button type="reset" class="btn btn-secondary btn-cancel" name="clear" id="clear" value="<?php echo xla('Clear'); ?>">
+                            <?php echo xlt('Clear'); ?>
+                        </button>
+                    </div>
+                </form>
+
+                <div class="table-responsive" id="immunization_list">
+                    <table class="table">
+                        <!-- some columns are sortable -->
+                        <tr>
+                            <th>
+                                <a href="javascript:top.restoreSession();location.href='immunizations.php?sortby=vacc';" title='<?php echo xla('Sort by vaccine'); ?>'>
+                                    <?php echo xlt('Vaccine'); ?>
+                                </a>
+                                <span class='small' style='font-family:arial'>
```

Comment:
Font family should not be changed here. Remove all the styling that changes the font family here.